### ISSUE TITLE
Copy PNG+XML when copying selection.

### ISSUE THREAD

**Issue by davidjgraph** [jgraph/drawio#1845]
When we copy, currently, we copy the data model of the selection. That only makes sense to our editor. To implement what you’re asking would require a seperate copy as image option.

I rather suspect nobody would naturally find the option, rather expect copy/paste to “just work”. There’s also the problem that people might copy as image into another diagram and then complain they can’t edit it.

Short version - both we and the user needs to understand what the paste target is.

The solution here is to change copy to create a PNG of the selection with the data model embedded within that PNG.

That would paste the right thing in both cases without exposing the underlying issue.

**Comment by alderg** [comment]
Not possible to add a ClipboardItem with a Promise (lazy) to create the image when requested, XML is removed from images when using image/png mime type and not possible to add multiple clipboard items for different mime types. Workaround via img tag with data URI that contains embedded XML and use image/html mime type for clipboard item is possible, but image creation cannot be delayed so will slow down copy operations in the app.
